tests/fixtures/schemas/customer_bad_evolution.avsc

```diff
@@ -1,27 +1,11 @@
 {
   "type": "record",
-  "name": "Customer",
+  "name": "Orders",
   "namespace": "examples",
   "fields": [
     {
-      "name": "name",
+      "name": "foo",
       "type": "string"
-    },
-    {
-      "name": "email",
-      "type": "string"
-    },
-    {
-      "name": "address",
-      "type": "string"
-    },
-    {
-      "name": "phone",
-      "type": "string"
-    },
-    {
-      "name": "age",
-      "type": "int"
     }
```

Comment:
There weren't any problems with this passing the invalid/incompatible schema evolution test, but I wanted to make the "bad evolution" path very clearly invalid and consistent between the three main schema types.